action.yml

```diff
@@ -15,9 +15,9 @@ runs:
     - name: Re-enable workflow
       env:
         GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
-      shell: bash
+      shell: sh
       run: |
-        [[ "${GITHUB_WORKFLOW_REF:?}" == "${GITHUB_REPOSITORY:?}"/.github/workflows/*.y?(a)ml@* ]]
+        expr "${GITHUB_WORKFLOW_REF:?}" : "${GITHUB_REPOSITORY:?}\/\.github\/workflows\/.*\.ya\?ml\@.*" > /dev/null
```

Comment:
`expr` is not a shell builtin, and we're not escaping the contents of `$GITHUB_REPOSITORY`, so any special regex characters in it keep their special meaning.

I think a better way to translate the `[[` bashism to posix sh is using `case`. Something like:
```sh
case "${GITHUB_WORKFLOW_REF:?}" in
  "${GITHUB_REPOSITORY:?}"/.github/workflows/*.y*ml@*) ;;
  *) false ;;
esac
```

It isn't particularly nice but that's not exactly surprising of POSIX sh :-)

Reply:
Yeah I missed the escaping part :) Thank you. Pushed the changes.